src/Parser.php

```diff
@@ -14,8 +14,13 @@ public function parse(string $filename)
         foreach ($lines as $line) {
             $line = trim($line);
             if ($line !== '') {
-                [$url, $status] = preg_split('/[\s]+/', $line);
-                $domains[] = new Domain($url, $status);
+                [$url, $status, $content] = preg_split('/[\s]+/', $line);
```

Comment:
Bit of a naive split here; doesn't account for content search with spaces, e.g. `http://httpbin.org/status/418 418 "teapot blah"`